dashboard/config/upstream-kcsan.config

```diff
@@ -8271,24 +8271,20 @@ CONFIG_DEBUG_PREEMPT=y
 # Lock Debugging (spinlocks, mutexes, etc...)
 #
 CONFIG_LOCK_DEBUGGING_SUPPORT=y
-CONFIG_PROVE_LOCKING=y
-# CONFIG_PROVE_RAW_LOCK_NESTING is not set
+# CONFIG_PROVE_LOCKING is not set
 # CONFIG_LOCK_STAT is not set
-CONFIG_DEBUG_RT_MUTEXES=y
-CONFIG_DEBUG_SPINLOCK=y
-CONFIG_DEBUG_MUTEXES=y
-CONFIG_DEBUG_WW_MUTEX_SLOWPATH=y
-CONFIG_DEBUG_RWSEMS=y
-CONFIG_DEBUG_LOCK_ALLOC=y
-CONFIG_LOCKDEP=y
-# CONFIG_DEBUG_LOCKDEP is not set
+# CONFIG_DEBUG_RT_MUTEXES is not set
```

Comment:
The previous commit did not enable these, these were enabled before:
https://github.com/google/syzkaller/commit/c9cef45b35aaf2050d438735b2178f86eab35100
is this intentional? wonder why do we have this hysteresis here...

Reply:
PROVE_LOCKING selects all these implicitly, so yes, it'd be intentional.

Reply:
Right, I was looking at a wrong commit. Here is the right one:
https://github.com/google/syzkaller/commit/02034dac31a9c4dfecfa163f445ef1921c442268